lib/calyx.rb

```diff
@@ -1,4 +1,7 @@
+require 'erb'
+
 module Calyx
+  #The Grammar class and the Production module was written by Mark Rickerby in 2015, and licensed under the MIT license.
```

Comment:
I’d much prefer it if the classes had descriptive comments for `yard` doc compatibility. Authorship can be inferred from looking at Git history.